PROCFILE

```diff
@@ -0,0 +1 @@
+web: python app.py
```

Comment:
To run the react app, I thought we need npm run build in Procfile?

Reply:
Do we? I didn't know that.

Reply:
Just checked, heroku automatically runs build scripts.